dask/values.yaml

```
@@ -69,6 +69,13 @@ worker:
   #    cpu: 1
   #    memory: 3G
   #    nvidia.com/gpu: 1
+  mounts: {}
```

Could we add a description comment here?

```suggestion
  mounts: {}  # Describe me!
```